harness/determined/common/experimental/determined.py

```diff
@@ -27,19 +61,61 @@ def __init__(
     ):
         self._session = Session(master, user)
 
+        host = self._session._master
+        if host[-1] == "/":
```

Comment:
sorry, `rstrip()` not `rtrim()`

Reply:
this `if` statement is no longer required; you can always run `host = host.rstrip("/")`